homeassistant/components/bluetooth/common_device.py

```diff
@@ -0,0 +1,24 @@
+"""Common functions related to Bluetooth device management."""
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+from homeassistant import const
+from homeassistant.helpers.entity import DeviceInfo
+
+if TYPE_CHECKING:
+    from sensor_state_data import SensorDeviceInfo
+
+
+def sensor_device_info_to_device_info(
```

Comment:
I'd probably keep the reference to `hass`
```suggestion
def sensor_device_info_to_hass_device_info(
```